server/src/packages/packages.service.ts

```diff
@@ -70,6 +70,31 @@ const PACKAGE_TYPE_OPTIONSET = {
 export class PackagesService {
   constructor(private readonly crmService: CrmService) {}
 
+  // Gets the lateset PAS Package on Project `projectId`
+  async getLatestPasForm(projectId): Promise<any> {
+    try {
+      const { records: pasPackages } = await this.crmService.get('dcp_packages', `
+        $filter=
+          _dcp_project_value eq ${projectId}
+          and
+          dcp_packagetype eq ${PACKAGE_TYPE_OPTIONSET['PAS_PACKAGE'].code}
+        &$expand=
+          dcp_pasform
+      `);
+
+      // if pasA is of a higher version than pasB, it should come first
+      const [{ dcp_pasform: latestPasForm }] = pasPackages.sort((pasA, pasB) => {
+        return pasB.dcp_packageversion - pasA.dcp_packageversion;
+      });
+
+      return latestPasForm;
+    } catch(e) {
+      const errorMessage = `Error finding a PAS Package on given Project. ${e.message}`;
+      console.log(errorMessage);
+      throw new HttpException(errorMessage, HttpStatus.UNAUTHORIZED);
+    }
+  }
+
   async getPackage(packageId) {
```

Comment:
this is starting to do way more than get a package. It gets a package, gets related
  forms, detects the form type, includes that form with the payload, includes additional
  munging that's specific to the form, and also includes attached documents.